src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,44 @@
+package calculator
+
+import java.util.Stack
+
+enum class Operator : Word {
```

Comment:
코틀린의 함수는 일급 함수입니다. 

일급함수는 객체로 취급되는 함수를 의미하며 다음과 같은 특징을 갖고 있습니다.

```
일급함수는 함수가 객체로 취급될 수 있습니다.
일급함수는 함수 객체를 인자로 넘길 수 있어야 합니다.
일급함수는 함수 객체를 인자로 넘길 수 있어야 합니다.
```

아래 코드처럼 수정해보는건 어떨까요?

```suggestion
enum class Operator(val operate: (Int, Int) -> Int) : Word {
```